src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -45,6 +65,20 @@ public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator
 
         // TODO использовать Arrays.sort
 
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person left, Person right) {
+                return left.getFirstName().compareTo(right.getFirstName());
+            }
+        });
+
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person left, Person right) {
+                return left.getLastName().compareTo(right.getLastName());
+            }
+        });
```

Comment:
Comparator.thenComparing() could be used instead of second Arrays.sort() - thus code will be more compact.